.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: $DOCKER_USERNAME/katex-fonts:29d4cdbc
```

Comment:
Does this docker image live on docker hub?  Why do we need environment variables for the user name?

Reply:
@kevinbarabash Yes. Using environment variables is for easier maintenance.